src/Sulu/Bundle/AdminBundle/Resources/js/containers/ViewRenderer/tests/ViewRenderer.test.js

```diff
@@ -15,9 +15,15 @@ test('Render view returned from ViewRegistry', () => {
     expect(viewStore.get).toBeCalledWith('test');
 });
 
-test('Render view returned from ViewRegistry with passed props', () => {
-    viewStore.get.mockReturnValue((props) => (<h1>{props.value}</h1>));
-    const view = render(<ViewRenderer name="test" parameters={{value: 'Test from props'}} />);
+test('Render view returned from ViewRegistry with passed router', () => {
+    const router = {
+        attributes: {
+            value: 'Test from props',
```

Comment:
check test value?

Reply:
Changed it to `Test attribute`. And the `URLSearchParams` are already decoding the spaces.